soundfile.py

```diff
@@ -1131,7 +1161,8 @@ def truncate(self, frames=None):
                               _ffi.new("sf_count_t*", frames),
                               _ffi.sizeof("sf_count_t"))
         if err:
-            raise RuntimeError("Error truncating the file")
+            err = _snd.sf_error(self._file)
```

Comment:
What is the purpose of this line?

Reply:
The 'err' returned by sf_command here is not a libsndfile error code, but just '-1', which cannot be used for the SoundFileError.  sf_error() gives a more useful value (currently SF_ERR_SYSTEM  in this case).

Reply:
Thanks for the info, that's interesting.

Should we add comment explaining this?